components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/query/SQLQuery.java

```diff
@@ -197,8 +199,22 @@ public void init(String query) throws DataServiceFault {
          * Create Calendar instance with "UTC" time zone
          * to use when setting timestamp for prepared statements
          * and retrieving timestamps from result sets
+         * This time conversion feature can be enabled/disabled
+         * using -Ddss.legacy.timezone.mode=true or -Ddss.legacy.timezone.mode=false.
+         * Default values are dss.legacy.timezone.mode=false with default timezone "UTC"
+         * This default time zone can be set via -Ddss.timezone=
          */
-        calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        String userTimeZone = System.getProperty(RDBMS.DSS_TIMERZONE);
+        if(userTimeZone == null || userTimeZone.isEmpty()) {
```

Comment:
Need a space after if

Reply:
https://stackoverflow.com/questions/46596508/wso2-dss-timezone-issue
can you please let me know how to fix this issue